### ISSUE TITLE
Is it possible to download llama-2 weights with this script?

### ISSUE THREAD

**Issue by ramkumarkoppu** [shawwn/llama-dl#27]
Is it possible to download llama-2 weights with this script? if yes, how to do it?

**Comment by ramsy0dev** [comment]
@ramkumarkoppu Yes it is possible to download the llama-2 weights with [this script](https://github.com/shawwn/llama-dl/blob/main/llama.sh), if you don't know how, then you can just follow the instructions mentioned in the  [download](https://github.com/shawwn/llama-dl#download) section in the [readme.md](https://github.com/shawwn/llama-dl/blob/main/README.md).

**Comment by ramkumarkoppu** [comment]
Is it not llama-1 weights by default? I am intrested in downloading more recent llama-2 weights

**Comment by ramsy0dev** [comment]
If you want to download llama-2 then you can just follow the instruction in this [official repo](https://github.com/facebookresearch/llama#download).
